packages/serverless-dev-runtime/lib/server.js

```diff
@@ -11,6 +11,7 @@ const { validateInputs } = require('./validation');
 const { getValidatedFunctionData } = require('./data');
 const { setupRoutes, updateRoutePaths } = require('./routes');
 const { createTemporaryFunction, cleanupArtifacts } = require('./files');
+// TODO - Find a way to replace this with @hubspot/cli-lib/lib/table without causing a circular dependency
```

Comment:
It looks like `table.js` is in `packages/cli` not `@hubspot/cli-lib`. Is that what causes the circular dependency?

Reply:
Thanks @gcorne, I guess I missed this. I moved table into `cli-lib` and removed it from both `cli` and `serverless-dev-runtime`. Both the cli and runtime work great. I think this is a more appropriate place for it anyway.